content/ja/docs/client-options.md

```diff
@@ -1,4 +1,37 @@
 ---
+title: ACME クライアント実装
 slug: client-options
-untranslated: 1
+top_graphic: 1
+lastmod: 2019-05-24
 ---
+
+{{< clientslastmod >}}
+
+Let's Encrypt は、与えられたドメインを制御する権限があなたにあることを検証し、証明書を発行するために、ACME プロトコルを使用しています。Let's Encrypt の証明書を取得するためには、使用する ACME クライアントを1つ選ぶ必要があります。
+
+以下に示す ACME クライアントはサードパーティにより提供されているものです。サードパーティ製クライアントは Let's Encrypt の制御下にはなく、レビューを行っているわけではないので、安全性や信頼性に対する保証をすることはできません。
+
+ブラウザ内で動作する ACME クライアントがいくつか存在しますが、以下のリストには掲載していません。その理由は、手動で更新するワークフローを推奨しているためです。手動での更新はユーザーエクスペリエンスを悪化させる原因であり、更新を忘れるリスクが高まります。
+
+# おすすめのクライアント: Certbot
+
+Let's Encrypt では、ほとんどの人に [Certbot](https://certbot.eff.org/) クライアントをおすすめしています。Certbot を使えば、証明書が簡単に取得でき、ユーザーの希望に応じてインストール作業も行ってくれるからです。使用するのが簡単で、多数のオペレーティングシステムで動作し、ドキュメントも充実しています。
+
+Certbot があなたが求める要件を満たさない場合や、単に他のクライアントを試してみたい場合でも、選択できるクライアントは以下に挙げるようにたくさんあります。リストは、実装言語や実行環境によってクループ分けをしています。
+
+{{< clients acme_v2="ACME v2 Compatible Clients" libraries="Libraries" projects="Projects integrating with Let’s Encrypt" >}}
```

Comment:
"Libraries" and "Projects integrating with Let’s Encrypt" can be translated

Reply:
I just fixed it!